html/rendering/non-replaced-elements/the-hr-element-0/hr.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<title>The hr element</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style>
+#ref {
+  display: block;
+  unicode-bidi: isolate;
```

Comment:
Thanks.

Maybe time to fix https://html.spec.whatwg.org/multipage/rendering.html#the-hr-element-2 then, because it's not mentioned there.